src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -236,12 +221,12 @@ public void returnsEmptyIteratorOnInvalidPartRequest() throws IOException {
     }
 
     /**
-     * RqMultipart.Base can return correct name set.
+     * RqMultipart.Fake can return correct name set.
      * @throws IOException If some problem inside
      */
     @Test
     public void returnsCorrectNamesSet() throws IOException {
-        final Request req = RqMultipartTest.request(
```

Comment:
@erimerturk I think we should delete method `request` from this test case, right?